Warnings when publishing Gradle plugin

Any chance we could get rid of those warnings:
```
> Task :compileJava
Note: /data/home/gsmet/git/quarkus-release/work/quarkus/devtools/gradle/src/main/java/io/quarkus/gradle/AppModelGradleResolver.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :javadoc
/data/home/gsmet/git/quarkus-release/work/quarkus/devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java:62: warning: no @param for uberJar
    public void setUberJar(boolean uberJar) {
```
?

--- a/devtools/gradle/src/main/java/io/quarkus/gradle/AppModelGradleResolver.java
+++ b/devtools/gradle/src/main/java/io/quarkus/gradle/AppModelGradleResolver.java
@@ -96,7 +96,7 @@ public class AppModelGradleResolver implements AppModelResolver {
             }
 
         }
-        return appArtifact.getPath();
+        return appArtifact.getPaths().getSinglePath();
     }
 
     @Override

--- a/devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java
+++ b/devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java
@@ -55,6 +55,7 @@ public class QuarkusBuild extends QuarkusTask {
     }
 
     /**
+     * @param uberJar Set to true if the build task should build an uberjar
      * @deprecated use {@code quarkus.package.type} instead
      */
     @Option(description = "Set to true if the build task should build an uberjar", option = "uber-jar")

